src/components/Property/Contacts/TenantsContactTable/TenantsContactTable.test.js

```diff
@@ -0,0 +1,59 @@
+import { render } from '@testing-library/react'
+import TenantsContactTable from './TenantsContactTable'
+
+describe('TenantsContactTable component', () => {
+  describe('when supplied with an empty tenants list', () => {
+    const tenants = []
+
+    it('renders text indicating that no tenants are available', () => {
+      const { asFragment } = render(<TenantsContactTable tenants={tenants} />)
+
+      expect(asFragment()).toMatchSnapshot()
```

Comment:
Wouldn't explicit assertions be better here than snapshot comparisons? Or good to have in addition to snapshot comparisons?

Reply:
I think for these static use cases, snapshot tests are better. They are much quicker to implement, and require less maintenance. 

Is there reasons why assertion tests would be better? Is there something im not testing?

Reply:
True, for these static cases snapshots make sense. I was probably thinking for more interactive cases.